Claroty/claroty-xdome/tests/test_smb_communication.json

```
@@ -0,0 +1,72 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "Claroty xDome",
+        "dialect_uuid": "9f47aa9f-52d7-4849-9462-cf7fc8bcd51a"
+      }
+    },
+    "message": "CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46|5|CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46|5|client_id=ZOLDKI0234 device_asset_id=NRLDV001 device_assignees=[] device_category=IT device_connection_type_list=['Ethernet'] device_ip_list=['1.2.3.4'] device_labels=[] device_mac_list=['fc:3f:db:0d:87:89'] device_manufacturer=HP device_model=EliteDesk 800 G2 DM 35W device_network_list=['Corporate'] device_note=None device_os=Windows 10 1607 device_site_name=Main device_subcategory=Computers device_type=PC device_uid=6162cd8a-8dc8-40b2-8a4a-e7a922862505 dst_ip=172.23.44.46 dst_mac=c8:d3:ff:9b:4f:9c dst_port=60383 event_description=Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46 event_id=11111111 event_timestamp=2022-02-06T14:21:52.873069+00:00 event_type=SMBv1 Communication msg_category=comm_event protocol=SMBv1 server_port=139 src_ip=1.2.3.4 src_mac=fc:3f:db:0d:87:89 src_port=139"
+  },
+  "expected": {
+    "message": "CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46|5|CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46|5|client_id=ZOLDKI0234 device_asset_id=NRLDV001 device_assignees=[] device_category=IT device_connection_type_list=['Ethernet'] device_ip_list=['1.2.3.4'] device_labels=[] device_mac_list=['fc:3f:db:0d:87:89'] device_manufacturer=HP device_model=EliteDesk 800 G2 DM 35W device_network_list=['Corporate'] device_note=None device_os=Windows 10 1607 device_site_name=Main device_subcategory=Computers device_type=PC device_uid=6162cd8a-8dc8-40b2-8a4a-e7a922862505 dst_ip=172.23.44.46 dst_mac=c8:d3:ff:9b:4f:9c dst_port=60383 event_description=Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46 event_id=11111111 event_timestamp=2022-02-06T14:21:52.873069+00:00 event_type=SMBv1 Communication msg_category=comm_event protocol=SMBv1 server_port=139 src_ip=1.2.3.4 src_mac=fc:3f:db:0d:87:89 src_port=139",
```

My bad. The sample is bad.
```suggestion
    "message": "CEF:0|Claroty|Industrial Device Security Platform|1.0|11111111|Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46|5|client_id=ZOLDKI0234 device_asset_id=NRLDV001 device_assignees=[] device_category=IT device_connection_type_list=['Ethernet'] device_ip_list=['1.2.3.4'] device_labels=[] device_mac_list=['fc:3f:db:0d:87:89'] device_manufacturer=HP device_model=EliteDesk 800 G2 DM 35W device_network_list=['Corporate'] device_note=None device_os=Windows 10 1607 device_site_name=Main device_subcategory=Computers device_type=PC device_uid=6162cd8a-8dc8-40b2-8a4a-e7a922862505 dst_ip=172.23.44.46 dst_mac=c8:d3:ff:9b:4f:9c dst_port=60383 event_description=Communication over SMBv1 was detected between 1.2.3.4 (PC) and 172.23.44.46 event_id=11111111 event_timestamp=2022-02-06T14:21:52.873069+00:00 event_type=SMBv1 Communication msg_category=comm_event protocol=SMBv1 server_port=139 src_ip=1.2.3.4 src_mac=fc:3f:db:0d:87:89 src_port=139",
```